index.js

```diff
@@ -1,11 +1,44 @@
 'use strict';
 const fs = require('fs');
+const os = require('os');
 const path = require('path');
 const childProcess = require('child_process');
 const runApplescript = require('run-applescript');
 const pify = require('pify');
 const rimraf = require('rimraf');
-const xdgTrashdir = require('xdg-trashdir');
```

Comment:
> It seems that to properly extend that module I'd have to break the signature of the export...

No, you just do `module.exports.all = function () { ...`. I do that all the time in my modules to add extra functionality.

Some of the code there could probably be shared too, since it's doing almost the same thing, just for different drives.